### PR TITLE
Filter name for creation time is called "started" instead of "created" #342

### PR DESCRIPTION
**Pull request by murthy95**
Addresses #342

### CODE REVIEW

File: go/pkg/param/filter.go [hunk 1]
```diff
@@ -150,7 +151,10 @@ where <operator> can be
 
 	// This is a hack, see https://github.com/replicate/replicate/issues/341
 	// "started" should probably be renamed/aliased to "created", see https://github.com/replicate/replicate/issues/342
-	if f.name == "started" {
+	if f.name == "started" || f.name == "created" {
+		if f.name == "started" {
+			console.Warn("Deprecated started use created instead")
```
Comment (andreasjansson) [inline go/pkg/param/filter.go:156]:
```suggestion
			console.Warn("The filter name 'started' is deprecated, please use 'created' instead")
```

File: go/pkg/param/filter.go [hunk 2]
```diff
@@ -150,7 +151,10 @@ where <operator> can be
 
 	// This is a hack, see https://github.com/replicate/replicate/issues/341
 	// "started" should probably be renamed/aliased to "created", see https://github.com/replicate/replicate/issues/342
```
Comment (andreasjansson) [inline go/pkg/param/filter.go:153]: we can remove this line now

**Review by andreasjansson — CHANGES_REQUESTED** [review]
This looks good, thank you! Could we also change the default sort in `addListSortFlag` in cli/list.go? And then run `make docs` from the go/ folder to update the documentation?

**Comment by bfirsh** [comment]
Thanks so much for the contribution @murthy95! You're also missing a sign-off on your commit, that certifies you're allowed to contribute to the project. If you click "Details" next to the failed DCO check, it has instructions on how to fix.

**Review by andreasjansson — APPROVED** [review]
Great, thanks again @murthy95 !

**Pull request merged**